packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Added
+  * Added optional personNameLevel prop to set the person name header level with default of 2.
+  * Added optional identifiersLongForm prop to add well defined descriptions for shortened/abbreviated identifiers.
+
+* Changed
+  * Changed patient nickname before/after CSS parenthesis property to instead be in the string.
```

Comment:
Also might add the functional reasoning/outcome for this change

Reply:
Added reasoning here https://github.com/cerner/terra-core/pull/3787/commits/3379e6c94a481f4aad6300f41cbf001ede650c42#diff-23b4f0bb288a072f5d6aa205a72db5a0affdcef4540c9d16ae1aa41e917e0461R10